src/persist-client/src/cache.rs

```diff
@@ -108,18 +108,19 @@ impl PersistClientCache {
                         consensus.clone().open()
                     })
                     .await;
+                let consensus =
+                    Arc::new(MetricsConsensus::new(consensus, Arc::clone(&self.metrics)));
+                self.rtt_latency_tasks.push(Arc::new(
```

Comment:
We could consider stashing these tasks in a consensus_uri->task map instead of a vec (or cramming it into the value of the existing blob / consensus maps) as a way to make it a little harder to accidentally create too many per backing store.

Reply:
done! (except moved them into the existing maps, mostly because I know #17440 is coming up and that structure will make my life easier)

this was essentially a rewrite of the PR, so please take another look (I've lost trust in myself on this bit of code)